src/app/campaign/campaign.component.ts

```diff
@@ -77,19 +82,26 @@ export class CampaignComponent implements OnInit, OnDestroy {
       )
       .subscribe(({ params, advertisers, accounts }) => {
         this.campaignStoreService.load(params.get('id'));
```

Comment:
This also still loads the campaign in the campaign store. I think we can live with that for now while we transition.